documentation/modules/ROOT/pages/configuration/topic-auto-create-config.adoc

```diff
@@ -0,0 +1,216 @@
+// Category: debezium-using
+// Type: assembly
+// ModuleID: configuring-debezium-to-auto-create-change-data-capture-topics
+// Title: Configuring {prodname} to use automatically create topics
+[id="cdc-topic-auto-create-config"]
+= Custom Topic Auto-Creation
+
+:toc:
+:toc-placement: macro
+:linkattrs:
+:icons: font
+:source-highlighter: highlight.js
+
+toc::[]
+
+{prodname} automatically creates *internal* topics for offsets, connector status, config
+storage and history topics. The destination topics for the captured tables will be
+automatically created with a default config by the Kafka brokers when
+`auto.create.topics.enable` is enabled.{empty} +
+When topic creation is disabled on the brokers, for example in production environments,
+or when the topics need a different configuration then these topics have to be created
+upfront either automated in a custom deployment process or manually until Kafka Connect 2.6.
+
+Since Kafka 2.6.0 Kafka Connect supports customizable topic auto-creation.
+
+== Kafka Connect
+
+Kafka Connect since Kafka 2.6.0 comes with topic creation enabled:
+
+[source,options="nowrap",shell]
+----
+topic.creation.enable = true
+----
+
+// TODO: how to express this for downstream?
+[NOTE]
+====
+If you don't want to allow automatic topic creation by connectors you can set this value to `false`
+in the Kafka Connect config (_connect-distributed.properties_ file or via environment variable
+_CONNECT_TOPIC_CREATION_ENABLE_ when using https://hub.docker.com/r/debezium/connect[Debezium's container image for Kafka Connect]).
+====
+
+== Configuration
+
+Topic auto-creation is based on groups. Every custom group has an `include` and an
+`exclude` property which are comma-separated lists of regular expressions matching
+topic names that should be included or excluded.
+
+[NOTE]
+====
+You can specify both, `include` and `exclude` parameters, but note that exclusion rules
+have precedent and override any inclusion rules for topics.
+====
+
+You don't have to specify any custom group. When there's no custom group registered or
+the registered group's `include` patterns don't match the topic which is to be created
+then the default config will be used.
+
+You can specify all https://kafka.apache.org/documentation/#topicconfigs[*topic level configuration parameters*]
```

Comment:
There's a var in antora.yml or similar for the Kafka docs base URL.

Reply:
Done